client/src/components/Form/Errors.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { colors } from 'models/constants';
+import { fonts } from 'models/styles';
+
+const Icon = ({ className }) => (
```

Comment:
Maybe I'm missing something, but how can this `Icon` component get passed a `className` here?

Reply:
It's passed by `styled-components`! (the `styled(Icon)` a little bit lower).

I've been meaning to abstract out the Icon components to take in a name that maps to the font awesome icon, but haven't gotten around to it. `ErrorIcon` is the actual one being used, which is the styled version of `Icon`

Reply:
Ah, I see